subsys/mgmt/mcumgr/smp_bt.c

```diff
@@ -64,10 +114,21 @@ static ssize_t smp_bt_chr_write(struct bt_conn *conn,
 	zephyr_smp_rx_req(&smp_bt_transport, nb);
 
 	return len;
+#endif
 }
 
 static void smp_bt_ccc_changed(const struct bt_gatt_attr *attr, uint16_t value)
```

Comment:
We seem to be using the ccc_changed callback to detect transport connection/disconnection. @MarekPieta @joerchan @cvinayak can we sanity-check this approach to be sure this is the correct one?

Reply:
Should check if this is called when bonded peers are disconnected. I think it is, but not 100% sure.